webapp/www/templates/game.html

```diff
@@ -0,0 +1,8 @@
+<ion-view view-title="Game">
+	<ion-content>
+		<ion-item>
+			<iframe src='http://54.159.38.241:3000/static/front' width="100%" height="550" frameborder="0" allowfullscreen>
```

Comment:
you want to use https://em-game.eecs.berkeley.edu here. Otherwise, the game won't load because of https <---> http mismatch. That's what happening for the trip planner on the current production.